scripts/issue/1380/aura_divest.py

```diff
@@ -0,0 +1,67 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+from rich.console import Console
+
+C = Console()
+
+# Contracts
+SAFE = GreatApeSafe(r.badger_wallets.treasury_voter_multisig)
+VAULT = r.badger_wallets.treasury_vault_multisig
+TROPS = r.badger_wallets.treasury_ops_multisig
+LOCKER = SAFE.contract(r.aura.vlAURA)
+GRAVIAURA = SAFE.contract(r.sett_vaults.graviAURA)
+
+# Tokens
+AURA = SAFE.contract(r.treasury_tokens.AURA)
+WETH = SAFE.contract(r.treasury_tokens.WETH)
+AURABAL = SAFE.contract(r.treasury_tokens.AURABAL)
+
+
+# artificially create slippage on the quoted price from cowswap
+COEF = 0.98
+# time after which cowswap order expires
+DEADLINE = 60 * 60 * 3
+
+
+def main(claim_rewards=False):
+    SAFE.init_cow(prod=True)
+
+    # Since the upKeep was shutdown, rewards must be manually claimed
+    rewards = LOCKER.claimableRewards(SAFE.account)[0][1]
+    if rewards > 0 and claim_rewards:
```

Comment:
could check `claim_rewards` and `unlockable` first considering "rewards can wait until next epoch" like:

```
if claim_rewards and unlockable > 0 and LOCKER.claimableRewards(SAFE.account)[0][1] > 0:
```